samples/security/spire/istio-spire-config.yaml

```diff
@@ -6,6 +6,8 @@ spec:
   profile: default
   meshConfig:
     trustDomain: example.org
+    defaultConfig:
+      holdApplicationUntilProxyStarts: true
```

Comment:
Could you add comment explaining what this config does and why you want it to be `true`?

Reply:
+1, just because it _can_ support it doesn't mean we should set it. I would recommend removing this, it just adds noise and isn't relevant to SPIRE

Reply:
done.I added some comment, please review.

Reply:
@maxlambrecht  @howardjohn  How about now?

Reply:
I'm ok with these changes.

Reply:
@howardjohn  hello,could you review it again?thanks.

Reply:
This hgas nothing to do with SPIRE so I would prefer to not have this since it adds confusion. But ok with it

Reply:
yes,it nothing to SPIRE, but it will influence the configuration of Spire-Agent,we need to add the disable_container_selectors = true if we want to use spire on istio.

Reply:
ok.I get it, I update this pr.

Reply:
@howardjohn hello,Is ok now?